live-examples/css-examples/basic-box-model/contain.html

```diff
@@ -16,11 +37,11 @@
 
 <div id="output" class="output large hidden">
     <section id="default-example" class="default-example">
-        <div class="example-container">
-            <div id="example-element" class="transition-all">
-                <div class="floating">Floating Element</div>
-                The Goldfish is a species of domestic fish best known for its bright colors and patterns.
-            </div>
+        <div id="example-element" class="card">
+            <h2>Card Title</h2>
```

Comment:
```suggestion
            <h2>Card Title</h2>
```
Should we call this "Containment element" to indicate that this is the element that the example is using to demo the 'contain' property? or "Element with 'contain'"

Reply:
done in a89beb1e31098f822275bd204bd5120c6ae134b6